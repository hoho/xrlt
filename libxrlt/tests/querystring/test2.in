ololo=%D0%B6%D0%BE%D0%BF%D0%B0
