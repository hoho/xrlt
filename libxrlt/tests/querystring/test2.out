<?xml version="1.0"?>
<root>
  <ololo>&#x436;&#x43E;&#x43F;&#x430;</ololo>
</root>
